test/chroner/administrative_test.exs

```diff
@@ -0,0 +1,141 @@
+defmodule Chroner.AdministrativeTest do
+  use ExUnit.Case, async: false
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+
+  import Chroner.Administrative
+
+  alias Chroner.Administrative.{Doctor, User}
+  alias OAuth2.Response
+
+  setup do
+    valid_client = Chroner.client(access_token: System.get_env("ACCESS_TOKEN"))
```

Comment:
Why `System.get_env`? Will this work if I don't have `ACCESS_TOKEN` configured?

Reply:
Yes, I should switch it to `fetch`.

Reply:
So that means I need to register an account at DrChrono, if I want to run these tests, right?

Reply:
My mistake. It needs to stay `get_env`, this way will work without `ACCESS_TOKEN` for existing cassettes but it won't work if you want to add new tests. In that case you muse provide token to run tests and create cassettes.